frontend/src/scenes/onboarding/onboardingLogic.tsx

```diff
@@ -258,8 +258,6 @@ export const onboardingLogic = kea<onboardingLogicType>([
             }
             if (step) {
                 actions.setStepKey(step)
-            } else {
-                actions.resetStepKey()
```

Comment:
Removing this fixes some weird bug in the `/products` onboarding flow where I have to click on "set up again" twice to get the next page to show up. I'm not _entirely_ sure why this works...

Here's a before video: 

https://github.com/PostHog/posthog/assets/21014901/1a8a50cc-8350-40d3-a7b1-11725aa4096c



And after video: 

https://github.com/PostHog/posthog/assets/21014901/5069f562-b7b7-4b81-b801-700477bc1549